modules/shared/src/main/scala/org/tessellation/schema/aci/StateChannelOutput.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.schema.aci
+
+import org.tessellation.schema.address.Address
+
+sealed trait StateChannelOutput {
+  val address: Address
+}
+
+object StateChannelOutput {
+  case class Snapshot(address: Address, bytes: Array[Byte]) extends StateChannelOutput
```

Comment:
I guess we will be trying to deserialize the bytes based on the Address, so we don't need custom types introduced by the state channel owner?

Reply:
That is the case.